### ISSUE TITLE
Update word list

### ISSUE THREAD

**Issue by vogella** [languagetool-org/languagetool#329]
https://github.com/marcoagpinto/aoo-mozilla-en-dict/blob/master/en_GB%20(Marco%20Pinto)%20-%202.30%20-%202015-11-01/en-GB.dic contains screenshot/S but Languagetools still reports it as error.

Can you update your wordlist?

**Comment by danielnaber** [comment]
@MikeUnwalla Maybe you'd like to make a decision about this? There are two different dictionaries, http://wordlist.aspell.net/dicts/ and https://github.com/marcoagpinto/aoo-mozilla-en-dict/. We need to decide which one to use, I can then hopefully help with any technical details.

**Comment by marcoagpinto** [comment]
Guys,

It would be great if you could use my version since it has 22K+ new words added in the past three years (state of the art).

Kevin Atkinson rarely updates his dictionaries and when he does, he only adds basic words.

I have suggested thousands of words for US/CA in his GitHub but he only used some dozens, so I have given up suggesting words to him.

**Comment by MikeUnwalla** [comment]
@marcoagpinto, Thanks for your explanation.

@danielnaber, I suggest that we use Marco's dictionary for the reason that he gives.

**Comment by danielnaber** [comment]
Then let's update the dictionary after the release of 3.4, as we're in feature freeze now.

**Comment by danielnaber** [comment]
I wanted to do this today, turns out it's quite complicated:

* the process documented in the Wiki isn't quite up-to-date
* `create_dict.sh` filters by `hunspell -d` which removes `doesn't` etc., so this call of hunspell needs to be removed I assume
* Making an update leads to changes in suggestions, some suggestions get quite a bit worse and I don't know yet why
* The suggestions are tested by the same code for US and GB English so we'll need to change the tests if there are two different sources for these dictionaries
* words like `transexual` and `calender` are now accepted - `transexual` instead of `transsexual` shows up in some dictionaries on the web, so I'm not sure. I don't see why `calender` should be included when in probably 99% of the uses it's a misspelling of `calendar`.

So in a nutshell: it's more complicated than I though and I don't know when I can finish this (note to self: local branch is `en-dict-update`).

**Comment by dpelle** [comment]
> I don't see why calender should be included when in probably 99% of the
> uses it's a misspelling of calendar

English grammar.xml has a rule id=CALENDER to detect the invalid use of rare word "calender" vs the more frequent word "calendar". That rule has no context. Its pattern is merely:

            <pattern>
                <token>calender</token>
            </pattern>

**Comment by danielnaber** [comment]
en-GB had been updated in September, today I've updated en-US and en-CA (using `make_en_us_dict.sh` and `make_en_ca_dict.sh`). I'll not also try to update en-AU (from http://extensions.openoffice.org/en/project/english-dictionaries-apache-openoffice)

**Comment by danielnaber** [comment]
en-AU has also been updated, for the others (en-ZA, en-NZ) I couldn't find updated versions, so I'll close this issue.
